### PR TITLE
Define the AMD64 unwind codes in their own header.

### PR DESCRIPTION
**Pull request by pgavlin**
This matches the CFI unwind codes and allows the JIT to be more
precise about its includes.

### CODE REVIEW

**Comment by pgavlin** [comment]
@BruceForstall @jkotas @russellhadley @dotnet/jit-contrib PTAL

File: src/inc/win64unwind.h [hunk 1]
```diff
@@ -0,0 +1,117 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef _WIN64UNWIND_H_
+#define _WIN64UNWIND_H_
+
```
Comment (BruceForstall) [inline src/inc/win64unwind.h]: This header file should also be under #if defined(_TARGET_AMD64_). Win64 would also apply to ARM64.

**Comment by BruceForstall** [comment]
LGTM

**Comment by pgavlin** [comment]
OS X builds appear to be having the same problem that the Ubuntu builds were having recently. That leg has passed before on this commit, so I'm going to go ahead and merge this.

cc @mmitche

**Comment by mmitche** [comment]
@pgavlin What was that issue?

**Comment by pgavlin** [comment]
> 09:25:30 ERROR: Unable to find a build for artifact copy from: dotnet_corefx/master/osx_release

See [the log](http://dotnet-ci.cloudapp.net/job/dotnet_coreclr/job/master/job/checked_osx_tst_prtest/839/console) for more details.

**Comment by mmitche** [comment]
@pgavlin Thanks, priya I thought merged the coreclr change for that yesterday.

**Pull request merged**